{"name": "Unsuccessful login", "status": "passed", "steps": [{"name": "Error message is visible", "status": "passed", "start": 1737859320994, "stop": 1737859320994}], "parameters": [{"name": "username", "value": "'standard_user'"}, {"name": "password", "value": "''"}, {"name": "driver", "value": "'Chrome'"}], "start": 1737859320826, "stop": 1737859321023, "uuid": "5d051aad-0153-41a9-bd6d-3021c20f0dde", "historyId": "b461054c34bef2ff4985443520df9a01", "testCaseId": "b876685be5aab7257716ff999cc789c0", "fullName": "Tests.test_Login.TestLogin#test_1", "labels": [{"name": "feature", "value": "Login"}, {"name": "severity", "value": "normal"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "fv-az1391-799"}, {"name": "thread", "value": "2019-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_Login"}]}